### DISCUSSION TITLE
how-to-experience-86-dos/

### DISCUSSION THREAD

**Discussion by giscus** [InstallerLegacy/neovirtualhub#1] [Comments]
# how-to-experience-86-dos/

86-DOS is a “quick and dirty” operating system clone of CP/M created by Seattle Computer Products for their 8086 S-100 systems. It was later bought by Microsoft and used as the basis for IBM PC-DOS and MS-DOS. Sadly, it cannot be emulated by any software we have installed so far. Thankfully, there exist an emulator, which is very tiny in size and we can run 86-DOS on it.

https://neo.virtualhub.eu.org/how-to-experience-86-dos/

**Comment by pulkitkrishna00** [comment]
Hello, This new website is nice!
